houston/ardu/sandbox.py

```diff
@@ -119,4 +120,4 @@ def _stop(self) -> None:
         if self.connection:
             self.connection.close()
         # close the SITL
-        self.bugzoo.command('killall5 -9 build/sitl/bin', stdout=False, stderr=False, block=False)
+        self.bugzoo.command("ps aux | grep -i sitl | awk {'print $2'} | xargs kill -9", stdout=False, stderr=False, block=False)
```

Comment:
I think this should be `block=False`, otherwise we could end up with a strange race condition.

Reply:
Do you mean `block=True`?

Reply:
I do mean that. Ha.